### DISCUSSION TITLE
Island Shop

### DISCUSSION THREAD

**Discussion by Tptuaazn** [BG-Software-LLC/SuperiorSkyblock2#194] [Denied Feature Requests] [answered]
I don't think it's a bad idea.

**Comment by OmerBenGera** [comment, marked as answer]
Hey, mind giving more information about the suggestion?
Is it gonna be similar to auction house, but somehow per island?

**Reply by Tptuaazn** [reply]
I have 2 suggestions.

1. Island Shop
The owner must manually set the items that will be in the store.

It should look like the store at ASkyblock

Commands: island shop<shops>
Settings: islandshops.yml

2. Player Shop
Hmm, It will be similar to AuctionHouse in some functions (buy, sell, retrieve sold items).

And players will be able to sell their own items. Like I said, it should be quite similar to the AuctionHouse.

Commands: island pshop<playershop> or only playershop<s>
Database file: playershopdata.db
Settings: playershop.yml (owner will set decoration item locations 🤔)

The item's location will be set there when sold.

Recommend: Island Shop

... support PlayerPoints, TokenManager

My explanation may not be clear, quite confusing ..

**Reply by OmerBenGera** [reply]
About the player shop - just use AuctionHouse, there's no need to implement an exact replica of another plugin.
About the island shop - mind giving more information of how it will work?

**Reply by Tptuaazn** [reply]
I think leaving it aside, using a certain shop plugin will solve this problem.
